stash_engine/lib/stash/event_data/citations.rb

```diff
@@ -8,53 +8,37 @@ class Citations
       include Stash::EventData
 
       attr_reader :doi
-      # This first domain worked earlier this week though with intermittant 500 errors
-      # DOMAIN = 'https://query.eventdata.crossref.org'.freeze
 
       # This domain they say is what I should use, but it returns blank strings and no json
-      BASE_URL = 'https://api.eventdata.crossref.org/v1/events'.freeze
+      # BASE_URL = 'https://api.test.datacite.org/events'.freeze
+      BASE_URL = 'https://api.datacite.org/events'.freeze
       EMAIL = 'scott.fisher@ucop.edu'.freeze
-      # only keep these relations from these sources, based on https://support.datacite.org/v1.1/docs/eventdata-query-api-guide
-      # DATACITE_INCLUDE = %w[is_cited_by is_supplement_to is_referenced_by is_compiled_by is_source_of is_required_by].freeze
-      # CROSSREF_INCLUDE = %w[cites is_supplemented_by compiles requires references].freeze
-      EXCLUDE_LIST = %w[has_version is_version_of is_new_version_of is_previous_version_of is_identical_to].freeze
+      DATACITE_URL = 'https://doi.org/'.freeze
+
+      OTHERS_CITING_ME = %w[cites describes is-supplemented-by references compiles reviews requires has-metadata documents
+                            is-source-of].freeze
+      ME_CLAIMING_CITATION = %w[is-cited-by is-supplement-to is-described-by is-metadata-for is-referenced-by
+                                is-documented-by is-compiled-by is-reviewed-by is-derived-from is-required-by].freeze
 
       def initialize(doi:)
-        @doi = doi
-        @doi = doi[4..-1] if doi.downcase.start_with?('doi:')
+        @doi = doi&.downcase # had lots of problems from DataCite eventdata with an upcase and DOIs are supposed to be case insensitive
+        @doi = doi[4..-1] if doi.start_with?('doi:')
         @base_url = BASE_URL
         @email = EMAIL
```

Comment:
You may want to change this to use the one defined in the app config rather than your personal address 🙃

Reply:
Thanks Brian.  I fixed this up to use the configured email instead of hard coding.